### ISSUE TITLE
Feature request: save temp files to a different folder

### ISSUE THREAD

**Issue by HommeBenny** [datawhores/OF-Scraper#223]
I am using Google Drive and all video temp files will be synced as soon as they are downloaded. Would be nice to have an option to save the temp files to a temp folder then move the final video files to the actual folder after merging.

**Comment by cjb900** [comment]
a better option would be to just pause google drive syncing when you run the scraper and when the scraper is done and there are no temp files then start google drive syncing again.

**Comment by HommeBenny** [comment]
@cjb900 Google Drive doesn't work like that... Even a file is hard deleted from the folder, Google Drive will still upload the file to the cloud and then delete it from the trash. I guess it has a cache folder somewhere...

**Comment by cjb900** [comment]
Well I've used Google drive in the past like that. If you pause the syncing it wont sync anything in the directories it has set to sync and once you restart it it will only sync what is currently in the directories you have set to sync. It won't sync stuff that isn't in those directories. So when you start the scraper and it's downloading stuff if there are part/temp files once the scraper is done and those part/temp are gone you just start Google drive again. So unless Google has changed something recently that's how it's always worked for me.

**Comment by datawhores** [comment]
This has been added in the master branch

You'll have to change the temp_dir value from None to the path you want
